seeding.js

```diff
@@ -0,0 +1,221 @@
+var db = require('./server/db.js');
+var restaurantsJson = require('./restaurant_ID.json');
+
+
+//USERS INFO
+var randNameFirst = [
+  'BrokenTableDiner',
+  'John',
+  'Jane',
+  'Chris',
+  'Albert',
+  'Leon',
+  'Claire',
+  'Sherry',
+  'Mike',
+  'Tim',
+  'David',
+  'Brad'
+];
+
+var randNameLast = [
+  '',
+  'Smith',
+  'Doe',
+  'Redfield',
+  'Kennedy',
+  'Wesker',
+  'B',
+  'W',
+  'C',
+  'M',
+  'S',
+  'A',
+  'T',
+  'R',
+  '420',
+  '2'
+];
+
+var userNameFull = () => {
+  var first = randNameFirst[Math.floor(Math.random() * (randNameFirst.length))];
+  var last = '';
+  if (first === 'BrokenTableDiner') {
+    return first + last;
+  } else {
+    last = randNameLast[Math.floor(Math.random() * (randNameLast.length))];
+    return first + last;
+  }
+};
+
+var randLocation = [
+  'Seattle',
+  'San Francisco',
+  'San Diego',
+  'Los Angeles',
+  'Las Vegas',
+  'New York Area',
+  'Baltimore',
+  'New England',
+  'Atlantic City',
+  'Washington DC',
+  'Philadelphia'
+];
+
+var userLocation = () => randLocation[Math.floor(Math.random() * (randLocation.length))];
+
+var userVipStatus = ['No', 'Yes'];
```

Comment:
Math.random() >= 0.5

⏫ evalutes to either true or false. Can be set to equal userVipStatus

Reply:
Or, really, userVip. You don't need the intermediary variable